src/status_im2/contexts/add_new_contact/events_test.cljs

```diff
@@ -2,6 +2,9 @@
   (:require [cljs.test :refer-macros [deftest is are]]
             [status-im2.contexts.add-new-contact.events :as core]))
```

Comment:
I was confused by the namespace alias `core` when reading the tests. I think it'd be better to follow the existing convention and alias as `events` (*leaf of the namespace "path"*).

Reply:
My thinking here is that `core` is like a short-hand for `code-under-test` (the ns being tested). I can switch to using `events`, no problem. Thanks for letting me know!

Reply:
Oh but this is a valuable convention to have in my experience. An official guideline would be rather nice.

Some programming languages or test runners have some conventions, like using the word `subject`, which is frequently used in Ruby to mean the *thing* under test.

For the namespace under test I like your suggestion now that I understand the reasoning. But I still think we should keep the current convention and only use this `core` alias if we can agree on a new guideline, because otherwise it can be confusing.

Reply:
done